music_assistant/providers/audible/audible_helper.py

```diff
@@ -449,54 +520,36 @@ async def _parse_audiobook(self, audiobook_data: dict[str, Any] | None) -> Audio
             authors=UniqueList(authors),
             narrators=UniqueList(narrators),
         )
+
+        # Set metadata
         book.metadata.copyright = audiobook_data.get("copyright")
         book.metadata.description = _html_to_txt(
             str(audiobook_data.get("extended_product_description", ""))
         )
-        book.metadata.languages = UniqueList([audiobook_data.get("language", "")])
+        book.metadata.languages = UniqueList([audiobook_data.get("language") or ""])
         book.metadata.release_date = audiobook_data.get("release_date")
+
+        # Set review if available
         reviews = audiobook_data.get("editorial_reviews", [])
-        if reviews:
-            book.metadata.review = _html_to_txt(reviews[0])
+        if reviews and reviews[0]:
+            book.metadata.review = _html_to_txt(str(reviews[0]))
+
+        # Set genres
         book.metadata.genres = {
-            genre.replace("_", " ") for genre in audiobook_data.get("platinum_keywords", "")
+            genre.replace("_", " ") for genre in (audiobook_data.get("platinum_keywords") or [])
```

Comment:
```suggestion
            genre.replace("_", " ") for genre in (audiobook_data.get("platinum_keywords", []))
```

Why not assign a default value in the getter ?
Or is there a chance that the item does exist in the dict but with a null value ?

Reply:
Yes exactly, it seams like the item can exist in all states: missing, existing: null, existing: value.

Reply:
yikes, that is awful. All clear then.